sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -75,6 +76,19 @@ private[sql] object JDBCRelation {
     }
     ans.toArray
   }
+
+  def getEffectiveProperties(
```

Comment:
this indentation is still a little funky, see https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide